docs/versions/content-patterns.md

```diff
@@ -0,0 +1,313 @@
+::::
+
+:::::
+
+You can edit all of the space settings you just defined at any time, except for the URL identifier.
+
+## Sibling bullets 
+
+**Use case:** Requirements, limits, other simple, mirrored facts.
+
+**Number to use:** Ideal is one per option (e.g. one per deployment type). You might consider nested bullets in limited cases.
+
+### Examples
+
+::::{tab-set}
+:group: one-two
+
+:::{tab-item} One
+:sync: one
+
+#### Required permissions
+
+* **Serverless**: `Admin` or equivalent
+* **Stack v9**: `kibana_admin` or equivalent
+
+:::
+:::{tab-item} Two
+:sync: two
+
+#### Create a space
+
+The maximum number of spaces that you can have differs by [what do we call this]: 
+
+* **Serverless**: Maximum of 100 spaces.
+* **Stack v9**: Controlled by the `xpack.spaces.maxSpaces` setting. Default is 1000.
+:::
+::::
+
+## Callouts
+
+**Use case:** Happy differences, clarifications
+
+Some sections don’t apply to, say, Serverless, because we manage the headache for you. Help people to feel like they’re not getting shortchanged with a callout.
+
+If there’s a terminology change or other minor change (especially where x equates with y), consider handling as a note for simplicity.
+
+### Examples
+
+* *In **Manage TLS certificates** section*:
+
+  :::{tip}
+  Elastic Cloud manages TLS certificates for you.
+  :::
+
+* *In a **Spaces** overview*:
+
+  :::{note}
+  In Stack versions 9.0 and lower, **Spaces** are referred to as **Places**.
+
+## Prose (inline)
+
+**Use case:** Clarifying or secondary information
+
+**Number to use:** ~ once per section (use your judgement)
+
+**When to use:** Cases where the information isn’t wildly important, but nice to know, or to add basic terminology change info to overviews
+
+Sometimes, you can just preface a paragraph with version info. 
+
+### Example
+
+If you're managing a Stack v9 deployment, then you can also assign roles and define permissions for a space from the **Permissions** tab of the space settings. 
+
+When a role is assigned to *All Spaces*, you can’t remove its access from the space settings. You must instead edit the role to give it more granular access to individual spaces.
+
+## Prose (explanatory paragraphs and sections)
+
+**Use case**: Differences with a "why"
+
+**When to use:** Comparative overviews
+
+Sometimes, a close explanation of the differences between things helps people to understand how something works, or why something behaves the way it does. Compare and contrast differences in paragraphs when the explanation helps people to use our features effectively.
+
+You also might do this to compare approaches between deployment types, when [sibling bullets](#sibling-bullets) aren't enough.
+
+### Example
+
+::::{tab-set}
+:group: one-two
+
+:::{tab-item} One
+:sync: one
+
+The way that TLS certificates are managed depends on your deployment type:
+
+In **self-managed Elasticsearch**, you manage both of these certificates yourself. 
```

Comment:
IMO folks might assume this means they should bold these phrases, which I don't think they should.
```suggestion
In self-managed Elasticsearch, you manage both of these certificates yourself. 
```

Reply:
mm, I like the bolding here so it's more scannable :2c: